File: codewars/Js/TwoSum.js

function twoSum1(numbers, target) {

    let arr = new Array(2);

    for (let i = 0; i <numbers.length ; i++) {
        for (let j = i+1; j <numbers.length ; j++) {
            if(numbers[i]+numbers[j] === target) {
                arr[0] = i;
                arr[1] = j;
                return arr;
            }
        }
    }
}

File: codewars/Java/TwoSum.java
public class TwoSum {
    public static int[] twoSum(int[] numbers, int target) {

        int[] arr = new int[2];

        for (int i = 0; i <numbers.length ; i++) {
            for (int j = i+1; j <numbers.length ; j++) {
                if (numbers[i]+numbers[j] == target) {
                    arr[0] = i;
                    arr[1] = j;
                    return arr;
                }
            }
        }
        throw new IllegalArgumentException("No two sum");
    }
}
